File: Patterns/C++/Solid_rhombus_pattern.cpp
#include <iostream>

// Solid rhombus pattern
/*
             *****
            *****
           *****
          *****
         *****

*/
using namespace std;

int main()
{

    int count = 10;

    for (int i = 1; i <= count / 2; i++)
    {
        for (int j = 1; j <= count; j++)
        {
            if (j > count / 2 - i && j <= count - i)
            {
                cout << "*";
            }
            else
            {
                cout << " ";
            }
        }
        cout << "\n";
    }

    return 0;
}

File: Patterns/Kotlin/SolidRhombusPattern.kt
// Solid rhombus pattern
/*
             *****
            *****
           *****
          *****
         *****

*/

fun main() {
    var count = 10

    for (i in 1..count / 2) {
        for (j in 1..count) {
            if (j > count / 2 - i && j <= count - i) {
                print("*")
            } else {
                print(" ")
            }
        }
        println()
    }
}
